docs/proposals/20230612-node-os-image-caching.md

```diff
@@ -60,57 +51,63 @@ any additional information provided beyond the standard proposal template.
 
 ### Goals
 
-1. A solution using the Node Prototype pattern which cache’s nodes’ OS image on a regular interval and replaces it as a new ‘golden image’.
-1. Faster horizontal scale outs
+1. A solution using the Node Prototype pattern which cache’s nodes’ OS image on a regular interval and replaces it as a new Machine or MachinePool template VM image
+1. Faster horizontal scale outs of applications
 1. Prevent security breaking issues on node bootup from security updates being required immediately
-1. We will know we’ve succeeded when we can benchmark speed increases and successful image changes
+
+We will know we’ve succeeded when we can benchmark speed increases and successful image changes.
 
 ### Non-Goals/Future Work
 
 1. Extend the functionality to Windows nodes
 1. Optimization for efficiency and scalability
-1. A more complicated method of selecting a good candidate node such as incorporating manual forced prototype creation, which would perform better but take research to find the optimal method. 
+1. A more complicated method of selecting a good candidate node such as incorporating manual forced prototype creation, which would perform better but take research to find the optimal method
 1. Optimization of the default time interval into a specific, best general default which would have to be researched
 
 ## Proposal
 
 ### User Stories
 
-#### As an operator of CAPZ, I want faster horizontal scaling on new node deployment
-
-People setting up CAPZ clusters will be able to have faster horizontal scaling because their Nodes’s OS images will be continually cached, and thus they will be able to avoid longer pull times and avoid having to create warm nodes and overprovision.
+#### As an operator, I want to be able to have faster horizontal scaling with pods running quicker when scheduled
 
-#### As an operator, I want to be able to be more inline with security compliance as I spin up nodes
-
-As an operator I would like to be able to have my node’s OS image cache for the ability to avoid security alerts, cache security updates and patches so that I’ll be more security compliant and up to date when I have to spin up a new node. Otherwise, flags will be raised that a node is out of date on boot up before it finishes patching and a temporary security risk will exist.
+As a cluster operator I want to bring up new nodes that already have the container images of the applications it will run pre-cached so that pods start running quicker when scheduled, resulting in faster horizontal scaling.
 
-#### As an operator, I want to be able to have pods run quicker when scheduled
+#### As an operator of CAPZ, I want to reduce monetary and environmental costs
```

Comment:
I think these personas are distinct, so this might be more applicable?
```suggestion
#### As an operator of clusters managed by CAPZ, I want to reduce monetary and environmental costs
```